### PR TITLE
(#198) deployment-report displays external db correctly

### PR DESCRIPTION
**Pull request by lasiva**

### CODE REVIEW

File: viya_ark_library/k8s/test_impl/response_data/resources_dataservers.webinfdsvr.sas.com.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        "spec": {
+            "databases": [
+                {
+                    "name": "SharedServices"
+                }
+            ],
+            "registrations": [
+                {
+                    "dataServiceName": "sas-platform-postgres",
+                    "host": "my-daily-default-flexpsql.postgres.database.azure.com",
```
Comment (sasjowood) [inline viya_ark_library/k8s/test_impl/response_data/resources_dataservers.webinfdsvr.sas.com.json:31]: I think this value should be made more generic. Something like `postgres.host.test`. At the very least, we should remove `azure` from the value.

File: deployment_report/model/utils/config_util.py [hunk 2]
```diff
@@ -292,6 +302,58 @@ def _get_db_info_v2(pgclusters: Dict) -> Dict:
...
+            db_data: Optional[Dict] = resource_definition.get_spec()
+
+            dbs: Dict = dict()
+
+            if not db_data:
+                continue
+
+            try:
+                dbs = {
+                    # Keys.DatabaseDetails.DBTYPE: _UNAVAIL_DB_,
```
Comment (sasjowood) [inline deployment_report/model/utils/config_util.py:336]: Comment can be removed.

File: deployment_report/model/utils/config_util.py [hunk 3]
```diff
@@ -292,6 +302,58 @@ def _get_db_info_v2(pgclusters: Dict) -> Dict:
...
+                    Keys.DatabaseDetails.DBNAME: db_data[_DATASERVER_DATABASES_KEY_][0][_DATASERVER_NAME_KEY_],
+                    Keys.DatabaseDetails.DBSSL: db_data[_PGCLUSTER_SSL_KEY_],
+                    Keys.DatabaseDetails.DBHOST: db_data[_DATASERVER_REGISTRATIONS_KEY_][0][_PGCLUSTER_HOST_KEY_],
+                    Keys.DatabaseDetails.DBPORT: db_data[_DATASERVER_REGISTRATIONS_KEY_][0][_PGCLUSTER_PORT_KEY_],
+                }
+            except KeyError:
+                dbs = {
+                    # Keys.DatabaseDetails.DBTYPE: _UNAVAIL_DB_,
```
Comment (sasjowood) [inline deployment_report/model/utils/config_util.py:344]: Comment can be removed.

**Review by sasjowood — CHANGES_REQUESTED** [review]

File: viya_ark_library/k8s/test_impl/response_data/resources_dataservers.webinfdsvr.sas.com.json (hunk 1 above)
Comment (lasiva) [reply viya_ark_library/k8s/test_impl/response_data/resources_dataservers.webinfdsvr.sas.com.json]: done

File: deployment_report/model/utils/config_util.py (hunk 2 above)
Comment (lasiva) [reply deployment_report/model/utils/config_util.py]: done

File: deployment_report/model/utils/config_util.py (hunk 3 above)
Comment (lasiva) [reply deployment_report/model/utils/config_util.py]: done

**Review by sasjowood — APPROVED** [review]

**Review by sasjowood — APPROVED** [review]

**Pull request merged**